argocon-demos/2026-dont-do-what-donny-dont-does-the-10-dos-and-500-donts-of-workflows-success/examples/tailscale-secure-ingress/README.md

```
@@ -0,0 +1,39 @@
+# Secure ingress to Argo Server using the Tailscale Kubernetes Operator
+
+This example shows two ways to configure secure access to Argo Server using the Tailscale Kubernetes Operator:
+- Standalone Ingress.
+- HA Ingress (which uses a [ProxyGroup](https://tailscale.com/docs/features/kubernetes-operator/how-to/cluster-ingress#high-availability) for high availability and is most suitable for production use cases). 
+
+##  Ingress over Tailscale
```

markdownlint-cli2 (0.21.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)